### DISCUSSION TITLE
Outlining and Updating in Plot Tree View vs Document View

### DISCUSSION THREAD

**Discussion by Valdus92** [vkbo/novelWriter#2134] [Help]
I tried making a whole outline right under the title page and transformed it all into separate documents and hierarchies. It worked great. 

At first I was confused with the whole transform business in the plot view (since the Document view transforms automatically), 

Once you get the hang of it, it works well. I was just thinking if there is anything more I need to know about transforming. Or even...

Why doesn't the Project View update like the Document View?

**Comment by vkbo** [comment]
> Why doesn't the Project View update like the Document View?

What do you define as "Project View" and "Document View"? The Project View is the tree view of document files, and it should update. What information is missing or not updating?
